tests/integration/errors/test_EH_THROW_errors_caught_exception.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+For a certain number of error types, PHP triggers an exception if EH_THROW is toggled on.
+Verify we don't record the error if the exception is thrown and caught.
+There should be no traced errors, error events, or errors attached to span events.
+*/
+
+/*SKIPIF
+<?php
+if (version_compare(PHP_VERSION, "7.0", "<")) {
+  die("skip: PHP < 7.0 not supported\n");
+}
```

Comment:
Is this skipif needed? PHPs < 7.0 are no longer supported and tests are not run on those PHPs.

Reply:
please see: [cc91192](https://github.com/newrelic/newrelic-php-agent/pull/876/commits/cc91192420f66b05f72c7891d084da0f169867ee)